src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  height: 406px;
+  width: 200px;
+  padding: 206px 16px 16px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: 19px 32px;
```

Comment:
Inline styles for images (like using `background-image` here) can make it harder to manage images in different states or responsive designs. Consider adding a class for the image or using an `img` tag with an `alt` attribute for accessibility.